sdk/openai/Azure.AI.OpenAI/src/Azure.AI.OpenAI.csproj

```diff
@@ -11,13 +11,20 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <GenerateAPIListing>true</GenerateAPIListing>
     <NoWarn>$(NoWarn);CS1591;AZC0012;AZC0102;CS8002;CS0436;AZC0112</NoWarn>
-    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
     <ImplicitUsings>enable</ImplicitUsings>
     <LangVersion>preview</LangVersion>
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(UseExternalLib)' == 'true'">
```

Comment:
It'd be good to leave a brief, explanatory `<!-- comment -->` here (also in the Directory.build.props, which could then accompany the commented-out setting to true with the information about when to uncommented it for local validation)

Reply:
I've refactored this a little and added more comments in another place for more context on why this was added. I will push out another update soon.